resources/timezones/map_data.txt

```diff
@@ -5564,7 +5564,7 @@
 52998|America/New_York
 52999|America/Mexico_City
 53|America/Havana
-54|America/Buenos_Aires
+54|America/Argentina/Buenos_Aires
```

Comment:
We use CLDR data for timezone identifiers and this mapping is working as they specify, so we kindly suggest you file an issue with the CLDR team here: https://unicode.org/cldr/trac/newticket
These are standard timezone representation, so please verify before filing an issue.